packages/client/src/client/OpenFin/apps/Launcher/components/LaunchButton.styled.ts

```diff
@@ -18,16 +18,20 @@ export const StyledButton = styled.button<{
   position: relative;
 
   border-radius: 4px;
-  background-color: ${({ iconHoverBackground, active }) =>
-    active ? iconHoverBackground : "inherited"};
+  background-color: ${({ theme, active }) =>
+    active
+      ? theme.newTheme.color["Colors/Background/bg-primary"]
+      : "inherited"};
```

Comment:
minor thing .. I think `"unset"` trumps `"inherited"` now in most cases as it means inherited or initial